lib/matplotlib/axes/_axes.py

```diff
@@ -2296,7 +2296,7 @@ def _parse_bar_color_args(self, kwargs):
             facecolor = mcolors.to_rgba_array(facecolor)
         except ValueError as err:
             raise ValueError(
-                "'facecolor' or 'color' argument must be a valid color or"
+                "'facecolor' or 'color' argument must be a valid color or "
                     "sequence of colors."
```

Comment:
Please also dedent this wrapped line, as it should be aligned with the above one.

Reply:
@QuLogic Happy to, will make that change now!

Reply:
@QuLogic Done!